### DISCUSSION TITLE
Copilot for Azure Data Studio

### DISCUSSION THREAD

**Discussion by igaliney** [community/community#17692] [Copilot Conversations]
Hi,

As I understand, Azure Data Studio is based on the VS Code or uses VS Code engine.
Is it possible to use Copilot with Azure Data Studio? 

I installed an extension in the ADS but can't authorize it in the GitHub account due to the following error: "Please upgrade VS Code. We've made some updates to our authentication process for VS Code and GitHub. Please upgrade to the latest release of VS Code (1.66+)".

Maybe you have some tricks like ""enable-proposed-api":["github.copilot"]" for the newer version of Copilot?

Thanks,

**Comment by ChristianSGoncalves** [comment]
I'm with the same problem

**Comment by ishazel** [comment]
Also wondering about this, it would be amazing to have CoPilot in ADS!

**Reply by ishazel** [reply]
I sure hope Microsoft's plan to Copilot all the things doesn't overlook ADS :(

**Comment by hrkeni** [comment]
This would be fantastic

**Comment by aealden** [comment]
Yes, please!

**Comment by nester-sky** [comment]
I'm also waiting for this feature

**Comment by nmbazima** [comment]
Also waiting received this error: Unable to install extension 'github.copilot' as it is not compatible with the current VS Code engine version '1.67.0'.

**Comment by franticallycodes** [comment]
PLEASE!

**Comment by palominoc** [comment]
This is a MUST!! they can't forgot about ADZ!!

**Comment by winzig** [comment]
FYI, there is some activity happening here: https://github.com/microsoft/azuredatastudio/issues/22977

**Comment by tymtam2** [comment]
The workaround is to install Microsoft's [mssql extension for Visual Studio Code](https://aka.ms/mssql-marketplace) and use VS Code until ADS is updated.

**Comment by tymtam2** [comment]
Azure Data Studio **Insiders** 1.45 (May 2023) supports Copilot:

![image](https://github.com/community/community/assets/1097613/a416eeb9-158e-48b8-96ad-096f371318e8)



Version: 1.45.0-insider (user setup)
Commit: f7bc38d552e2cfbd7e8cadcf6cb40ed5f7cf9066
Date: 2023-05-09T05:59:52.851Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

I installed from 
https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch

**Reply by ishazel** [reply]
Thank you ADS team!
